forward_engineering/helpers/viewHelper.js

```diff
@@ -120,7 +121,7 @@ module.exports = {
 			const columnsNames = getColumnNames(_)(collectionRefsDefinitionsMap, columns);
 
 			if (fromStatement && columnsNames?.length) {
-				script.push(`AS SELECT ${columnsNames.join(', ')}`);
+				script.push(`AS SELECT ${columnsNames.join(',\n')}`);
```

Comment:
We might have an issue with a trailing comma in case the last column is commented out.
Not 100% sure if that will result in a syntax error, but it's better to cover this edge case.